.github/workflows/node-ci.yml

```diff
@@ -48,6 +43,16 @@ jobs:
         run: yarn install
       - name: yarn build:all
         run: yarn build:all
```

Comment:
I think we can remove this now - since the Compressed size action is doing another build anyway!